src/services/fileAccessControl.js

```diff
@@ -1,4 +1,4 @@
-import { get, isObject } from 'lodash';
+import _, { get, isObject } from 'lodash';
```

Comment:
Importing all of lodash is no issue, just would rather be consistent. In large files,`_.whatever` is a useful syntax to know where the method comes from. In either case, consistency within a file at least